presto-docs/src/main/sphinx/release/release-0.286.rst

```diff
@@ -0,0 +1,88 @@
+* Improve predicate pushdown through Joins. :pr:`21353`
+* Improve the readability of query plans by formatting numbers with commas for easier interpretation. :pr:`21486`
+* Add additional linear regression functions like REGR_AVGX, REGR_AVGY, REGR_COUNT, REGR_R2, REGR_SXX, REGR_SXY, and REGR_SYY. :pr:`21630`
+* Add UPDATE sql support in Presto. :pr:`21435`
+* Add a session property ``skip_hash_generation_for_join_with_table_scan_input`` to skip hash precomputation for join when the input is table scan, and the hash is on a single big int and is not reused later. The property defaults to not enabled. :pr:`20948`
+* Add a config property ``khyperloglog-agg-group-limit`` to limit the maximum number of groups that ``khyperloglog_agg`` function can have. The query will fail when the limit is exceeded. (The default is 0 which means no limit). :pr:`21510`
+* Add a feature config property ``limit-khyperloglog-agg-group-number-enabled`` to control whether to limit the number of groups for aggregation functions using KHyperLogLog state. :pr:`21824`
+* Add session property ``rewrite_expression_with_constant_expression`` which defaults to enabled. This optimizes queries which have an equivalence check filter or constant assignments. :pr:`19836`
+* Add session property ``rewrite_left_join_array_contains_to_equi_join`` that transforms left joins with an ARRAY CONTAINS condition in the join criteria into an equi join. :pr:`21420`
+* Add an option in the Presto client to disable redirects. :pr:`21024`. This fixes advisory [GHSA-xm7x-f3w2-4hjm](https://github.com/prestodb/presto/security/advisories/GHSA-xm7x-f3w2-4hjm)
+* Improve prestodb/presto docker image by including ``config.properties`` and ``jvm.config`` files. :pr:`21384`
+* Upgrade ``hadoop-apache2`` to ``2.7.4-12``.  This fixes errors like ``library not found: /nativelib/Linux-aarch64/libhadoop.so`` when running presto on ARM64.  :pr:`21483`
+* * Add validation in Presto client to ensure that the host and port of the next URI do not change during query execution in Presto, enhancing security by preventing redirection to untrusted sources. :pr:`21101`
+* Add :doc:`Ecosystem </ecosystem/list>` documentation. :pr:`21698`
+* Added 1 line in sphinx/index.rst. ---------------------------xxx---------
+* Add ``cte_hash_partition_count`` session property to specify the number of buckets or writers to be used when using CTE Materialization. :pr:21625
+* Add :doc:`/installation/deploy-helm` to Installation documentation. :pr:`21812`
+* Remove redundant sort columns from query plans if a unique constraint can be identified for a prefix of the ordering list. :pr:`21371`
+* Add changelog table ``$changelog`` that allows users  to track when records were added or deleted in snapshots. :pr:`20937`
+* Add `reservoir_sample<../functions/aggregate.html#reservoir_sample>`_ aggregation function which is useful for generating fixed-size samples. :pr:`21296`
```

Comment:
```suggestion
* Add `reservoir_sample <../functions/aggregate.html#reservoir_sample>`_ aggregation function which is useful for generating fixed-size samples. :pr:`21296`
```
Add space between `reservoir_sample` and `<` for the link to build correctly. Verified in local build.